org.tuxfamily.XMoto.appdata.xml

```diff
@@ -27,6 +28,10 @@ SentUpstream: 2014-09-25
     </p>
   </description>
   <url type="homepage">http://xmoto.sourceforge.net/</url>
+  <categories>
```

Comment:
Again don't need these, they go in the `.desktop` file.

Reply:
AppStream and desktop files serve different purpose though. There's a valid usecase for both and majority of the applications currently on flathub have category data in their appstream data.

Reply:
Appstream extracts info from desktop files at build time.

Reply:
In that case then I will remove the categories.